src/client/pfs/pfs.proto

```diff
@@ -312,12 +312,20 @@ message TagObjectRequest {
   repeated Tag tags = 2;
 }
 
+message ListObjectsRequest {}
+
+message ListObjectsTaggedWithPrefixRequest {
+  string prefix = 1;
+}
```

Comment:
Nvm, these are actually substantially different methods, one's for listing with objects the other's for listing with tags. Could we just call the second one `ListTag`?